src/component/mxgraph/shape/EndEventShape.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { MxGraphFactoryService } from '../../../service/MxGraphFactoryService';
+import { mxgraph } from 'ts-mxgraph';
+
+const mxEllipse: typeof mxgraph.mxEllipse = MxGraphFactoryService.getMxGraphProperty('mxEllipse');
+
+export default class EndEventShape extends mxEllipse {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public constructor(bounds: mxgraph.mxRectangle, fill: any, stroke: any) {
+    super(bounds, fill, stroke, 5);
```

Comment:
:question: are we sure we want to put style configuration in the code?
For future extension, this means that if I want to change the stroke width, I will have to extend the `EndEventShape` instead of changing the style. In my mind, `shapes` are not intended for such an usage.
Let's talk of that

Reply:
We can have the storke width as paramter, and add a default value as 5, if we don't want to extend it.
I think if all the default configuration is on the shape, it would be easier to define all the styles in just one method.

Maybe, we can see that after adding other shapes.

Reply:
> Maybe, we can see that after adding other shapes.

:+1:

Reply:
After reflexion, I agree with you to not mix style & shape. The stroke witdh is about style, not form.

Reply:
ok, so I am changing the shape by adding a `strokewidth` constructor argument with default value as suggested by @aibcmars